.github/workflows/build-release.yml

```diff
@@ -1,13 +1,10 @@
-name: Publish release binaries
+name: Build binaries and publish release
```

Comment:
Is it possible to either rename the workflow based on whether the tag is set, or import a workflow into another one (i.e. have the tested part named something else, and this name used for a workflow that delegates to the tested workflow and then uploads)? I want the name "Publish release binaries" to be kept for the workflow that does actually publish the release binaries. If neither option is possible, then please leave this named as-is.